packages/slider/src/constants.js

```diff
@@ -0,0 +1,6 @@
+export const sliderTypes = Object.freeze({
```

Comment:
We will want to `export` these `constants` in the `index.js` file so users have access to these and will know what `variants` are available.

Reply:
Thanks and updated!